tests/test_selection/test_recursive_feature_addition.py

```diff
@@ -0,0 +1,126 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.exceptions import NotFittedError
+from sklearn.linear_model import LinearRegression
+from sklearn.tree import DecisionTreeRegressor
+
+from feature_engine.selection import RecursiveFeatureAddition
+
+
+def test_classification_threshold_parameters(df_test):
+    X, y = df_test
+    sel = RecursiveFeatureAddition(
+        RandomForestClassifier(random_state=1), threshold=0.001)
+    sel.fit(X, y)
+
+    # expected result
+    Xtransformed = pd.DataFrame(X[['var_7', 'var_10']].copy())
+
+    # expected ordred features by importance, from most important
+    # to least important
+    ordered_features = ['var_7', 'var_4', 'var_6', 'var_9', 'var_0',
+                        'var_8', 'var_1', 'var_10', 'var_5', 'var_11', 'var_2', 'var_3']
+
+    # test init params
+    assert sel.variables == [
+        "var_0",
+        "var_1",
+        "var_2",
+        "var_3",
+        "var_4",
+        "var_5",
+        "var_6",
+        "var_7",
+        "var_8",
+        "var_9",
+        "var_10",
+        "var_11",
+    ]
+    assert sel.threshold == 0.001
+    assert sel.cv == 3
+    assert sel.scoring == "roc_auc"
+    # test fit attrs
+    assert np.round(sel.initial_model_performance_, 3) == 0.997
+    assert sel.selected_features_ == ['var_7', 'var_10']
+    assert list(sel.performance_drifts_.keys()) == ordered_features
+    # test transform output
```

Comment:
should we check the performance_drifts_ values as well or is this an overkill?

Sale with feature_performance_, should we test the values?